### PR TITLE
Deprecate Dockerfile.fuse and modify docs

### LINKED ISSUE #13742: Using an all-in-one image in whole alluxio
**Is your feature request related to a problem? Please describe.**

To reduce the related images of Alluxio.

**Describe the solution you'd like**

Build an all-in-one image

**Describe alternatives you've considered**

Combine the dockerfile into one.

**Urgency**

Normal

**Additional context**

No

### PR DESCRIPTION
**Pull request by ssz1997**
### What changes are proposed in this pull request?

Dockerfile.fuse has been integrated into Dockerfile. It is verified that the functionality is not deprecated. Corresponding docs are updated.

### Why are the changes needed?

Reducing the Alluxio related docker images. Simplify the docker image build process and ease user pain point in deploying alluxio related services in docker env.

Fixes #13742

### CODE REVIEW

**Review by LuQQiu — COMMENTED** [review]
@ssz1997 PTAL the related kubernetes scripts, alluxio-fuse docker image in those files also need to be changed to alluxio docker image. 

e.g. 
```
integration/kubernetes/helm-chart/alluxio/README.md:| `image` | Docker image | `alluxio/alluxio-fuse` |
integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml 
        - name: alluxio-fuse
          image: {{ .Values.fuse.image }}:{{ .Values.fuse.imageTag }}
integration/kubernetes/helm-chart/alluxio/values.yaml:  image: alluxio/alluxio-fuse
```
PTAL https://github.com/Alluxio/alluxio/pull/13234#pullrequestreview-645679404 and @jiacheliu3 will have more information

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
@Xenorith FYI,  check what else you want to disable internally after this PR merged

**Comment by ssz1997** [comment]
Sorry about all the commits caused by the email problem. I fixed the ones @LuQQiu mentioned above, but there might be more of it? I haven't done a comprehensive check.

**Review by apc999 — APPROVED** [review]
Thanks for the improvement

**Comment by apc999** [comment]
alluxio-bot, merge this please

File: integration/kubernetes/helm-chart/alluxio/values.yaml [hunk 1]
```diff
@@ -290,7 +290,7 @@ fuse:
     # Extra environment variables for the fuse pod
     # Example:
     # JAVA_HOME: /opt/java
-  image: alluxio/alluxio-fuse
+  image: alluxio/alluxio
```
Comment (LuQQiu) [inline integration/kubernetes/helm-chart/alluxio/values.yaml:293]: @ssz1997 i will prefer to remove the `.Values.fuse.image` and `.Values.fuse.imageTag`
and change all the places that using the `.Values.fuse.image` and `.Values.fuse.imageTag` to  {{ .Values.image }}:{{ .Values.imageTag }} for easier to maintain in the future. Can you help create a new PR for these changes?

integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml 
        - name: alluxio-fuse
          image: {{ .Values.fuse.image }}:{{ .Values.fuse.imageTag }}

Comment (ssz1997) [reply integration/kubernetes/helm-chart/alluxio/values.yaml:293]: Makes sense. I'll do it today.

**Comment by Xenorith** [comment]
i grepped for alluxio-fuse from the repo root and found some more references to alluxio-fuse:
- the README file in `integration/docker/README.md` has instructions about building the alluxio-fuse image; this should probably be removed
- there is a Dockerfile in `integration/csi/Dockerfile` that builds from the alluxio-fuse image. do we have any idea how this should be migrated to use the remaining Dockerfile?
- there are chinese docs that still have lots of references to the alluxio-fuse image. i'm assuming we are leaving them alone for now and will update them as needed in the future

**Pull request merged**